src/intl/en/page-upgrades-index.json

```
@@ -140,8 +122,8 @@
   "page-upgrades-question-10-answer-2": "But if you've followed the discussions, here's how the upgrades fit into technical roadmaps, and a bit on how they're changing.",
   "page-upgrades-question-10-answer-3": "Phase 0 described the work to get the Beacon Chain live.",
   "page-upgrades-question-10-answer-5": "Phase 1 originally focused on implementing the shard chains, but prioritization has shifted to 'The Merge' which is the next planned upgrade.",
-  "page-upgrades-question-10-answer-6": "Phase 1.5 was originally planned to follow shard implementations, when Mainnet would be added as the last shard to the Beacon Chain. To expedite the transition away from proof-of-work mining, Mainnet will instead represent the first shard to connect with the Beacon Chain. This is now known as 'The Merge' and will be a significant step towards a greener Ethereum.",
-  "page-upgrades-question-10-answer-7": "Though the plans around Phase 2 have been a point of intense research and discussion, with The Merge planned before shard chains, this will allow for continued reassessment as to the needs of Ethereum development moving forward. Given a rollup-centric roadmap, the immediate necessity of shard chains is debatable.",
+  "page-upgrades-question-10-answer-6": "Phase 1.5 was originally planned to follow shard implementations, when Mainnet would be added as the last shard to the Beacon Chain. With rollup technology progressing, the transition away from proof-of-work was expedited. Mainnet will instead be 'merged' into the Beacon Chain and remain as the execution layer of Ethereum. This will be a significant step towards a greener Ethereum.",
+  "page-upgrades-question-10-answer-7": "The plans around Phase 2 have been a point of intense research and discussion. With The Merge planned before sharding, and the advancements in layer 2 solutions, plans have shifted to provide a more simplified form of data sharding to maximize rollup efficiency. Current layer 2s are enabling the ability to scale transaction execution, and data sharding will help make storing the proofs for this data on layer 1 in a cheap and scalable manner. This will continue to undergo reassessment and research as progress is made.",
```

Not really sure what the final sentence here refers to. Could we cut it?


With last sentence:
```suggestion
  "page-upgrades-question-10-answer-7": "The plans around Phase 2 have been a point of intense research and discussion. With The Merge planned before sharding, and the advancements in layer 2 solutions, goals have shifted to provide a more simplified form of data sharding to maximize rollup efficiency. Current layer 2s enable the ability to scale transaction execution, and data sharding will allow for storage of proofs for this data on layer 1 in a cheap and scalable manner. This will continue to undergo reassessment and research as progress is made.",
```

Without:
```suggestion
  "page-upgrades-question-10-answer-7": "The plans around Phase 2 have been a point of intense research and discussion. With The Merge planned before sharding, and the advancements in layer 2 solutions, goals have shifted to provide a more simplified form of data sharding to maximize rollup efficiency. Current layer 2s enable the ability to scale transaction execution, and data sharding will allow for storage of proofs for this data on layer 1 in a cheap and scalable manner.",
```